src/sidebar/components/AutocompleteList.js

```diff
@@ -53,14 +53,14 @@ export default function AutocompleteList({
         // The parent <input> field should capture keyboard events
         // eslint-disable-next-line jsx-a11y/click-events-have-key-events
         <li
-          key={`autocomplete-list-${index}`}
+          key={`AutocompleteList-${index}`}
```

Comment:
The key could just be `index` here, since it only needs to be unique amongst its siblings.